crates/loader/src/bindle/assets.rs

```diff
@@ -76,18 +79,29 @@ impl Copier {
             to.display()
         );
         fs::create_dir_all(to.parent().expect("Cannot copy to file '/'")).await?;
-        let buf =
-            self.reader.get_parcel(&p.sha256).await.with_context(|| {
-                anyhow!("Failed to fetch asset parcel '{}@{}'", self.id, p.sha256)
+        let mut stream = self
+            .reader
+            .get_parcel_stream(&p.sha256)
+            .await
+            .with_context(|| anyhow!("Failed to fetch asset parcel '{}@{}'", self.id, p.sha256))?;
+
+        let mut file = fs::File::create(&to).await?;
```

Comment:
Could we add a `with_context(...)` here so that the user can see where the failure was and which file/parcel it occurred on?

Reply:
Good catch -- done.

Clippy also caught a bug, so I fixed that, too.  Note to self: `for x in stream.try_next().await? { .. }` almost certainly does not do what you want it to do.